src/Symfony/Bridge/Monolog/Tests/Processor/SwitchUserTokenProcessorTest.php

```diff
@@ -26,8 +28,13 @@ class SwitchUserTokenProcessorTest extends TestCase
 {
     public function testProcessor()
     {
-        $originalToken = new UsernamePasswordToken('original_user', 'password', 'provider', ['ROLE_SUPER_ADMIN']);
-        $switchUserToken = new SwitchUserToken('user', 'passsword', 'provider', ['ROLE_USER'], $originalToken);
+        if (class_exists(InMemoryUser::class)) {
+            $originalToken = new UsernamePasswordToken(new InMemoryUser('original_user', 'password', ['ROLE_SUPER_ADMIN']), 'provider', ['ROLE_SUPER_ADMIN']);
+            $switchUserToken = new SwitchUserToken(new InMemoryUser('user', 'passsword', ['ROLE_USER']), 'provider', ['ROLE_USER'], $originalToken);
+        } else {
+            $originalToken = new UsernamePasswordToken(new User('original_user', 'password', ['ROLE_SUPER_ADMIN']), null, 'provider', ['ROLE_SUPER_ADMIN']);
+            $switchUserToken = new SwitchUserToken(new InMemoryUser('user', 'passsword', ['ROLE_USER']), null, 'provider', ['ROLE_USER'], $originalToken);
```

Comment:
I don't this that this works if the `InMemoryUser` class does not exist.

Reply:
:facepalm: I'm bad in hiding my copy-paste workflow